data/schema/v1/Decision_Point-1-0-1.schema.json

```
@@ -41,23 +47,32 @@
 		},
 		"namespace": {
 		    "type": "string",
-		    "description": "Namespace (a short, unique string): For example, \"ssvc\" or \"cvss\" to indicate the source of the decision point"
+		    "description": "Namespace (a short, unique string): For example, \"ssvc\" or \"cvss\" to indicate the source of the decision point. See SSVC Documentation for details.",
+		    "pattern": "^[a-z0-9-]{3,4}[a-z0-9/\\.-]*$",
+		    "examples": ["ssvc", "cvss", "ssvc-jp", "ssvc/acme", "ssvc/example.com"]
 		},
 		"version": {
 		    "type": "string",
-		    "description": "Version (a semantic version string) that identifies this object"
+		    "description": "Version (a semantic version string) that identifies this object",
+	    "pattern": "^(0|[1-9]\d*)\\.(0|[1-9]\d*)\\.(0|[1-9]\\d*)(?:-((?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\\.(?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\\+([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?$"
```

Sorry, I missed to quote two `\`
```suggestion
	    "pattern": "^(0|[1-9]\\d*)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)(?:-((?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\\.(?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\\+([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?$"

---

comma missing at the end as well that is fixed now in the https://github.com/CERTCC/SSVC/pull/704/commits/0dc08d0092f8a9d49c33ecff5eb9499cf597bf84

---

I'd be most comfortable with the second example provided by 

https://semver.org/

specifically

https://regex101.com/r/vkijKf/1/

on the basis that if the semver.org thinks that's the regex, I'd take their word for it.

---

What we have is exactly that with the double backslashes for usage in the JSON schema that's all.

---

> Should we stick to [0-9] instead of \d as \d matches say Arabic numerals when regex is used with unicode?

I'd like that as it eliminates the ambiguousness. `\d` is unfortunately sometimes interpreted differently...